qiita_pet/support_files/doc/source/processingdata/deblur_2021.09.rst

```diff
@@ -0,0 +1,68 @@
+deblur version 2021.09
+======================
+
+The deblur version 2021.09 addresses a bug with the fragment insertion parsing and
+cache that ignored some fragments for getting an accurate placement in the tree. In
+summary, in some occasions SEPP will return multiple fragments in a single entry; which
+was unexpected by the qp-deblur plugin parser, which assumed only one entry - the
+extra features will be seen as missing by the plugin and that information was
+sent and stored in the cache provided by Qiita, then propagated to future studies and
+meta-analyses.
+
+This bug was resolved in this `pull request <https://github.com/qiita-spots/qp-deblur/pull/60>`__.
+
+It is important to note that this bug only applies to the fragments inserted into the tree, which is
+only part of the `deblur reference hit table`.
+
+How do I know if my study processing had this bug?
+----------------------------------------------------
+
+The easiest is to check the table summary reported `Number rejected fragments`. If the number is
+different between the qp-deblur v1.1.0 and qp-deblur v2021.09 then your study had this bug. To
+see the table summary, you need to navigate to the processing graph, click on the
+`deblur reference hit table` artifact and see the table summary.
+
+
+Sample counts implications
+--------------------------
+
+At the time of writing of this documentation Qiita had 978,052 16S deblured private or pubic samples.
+In the figure below, we have at different trimming lengths how samples we will recover
+based on the minimum number of sequences per sample - this is an important consideration
+as we normally need to remove samples below a given threshold for beta diversity
+calculations (via rarefactoin) or differential abundance testing.
+
+.. figure::  deblur2021.09_private_public.png
+   :align:   center
+
+A few conclusions from this plot:
+
+- The maximum number of samples that we will recover are 6,771 at `Trimming (length: 150)`
+  and min_seqs of 1,500; which represents a 0.7% increment in private and public samples.
+- At all Trimming lengths the curve tends to go up and then down based on min_seq,
+  which is a common trend seen in rarefacion plots
+
+How perversive is this bug?
+---------------------------
+
+For a better assessment we are going to focus on only on the 150 bps trimming length. At
+the time of writing of this documentation Qiita had 1,484 16S preparations that have a
+150 bps deblur table. Of those:
+
+- 96.6% of preparations had 0-10% of features lost
+- 12.6% had 10-20% of the features lost
+-  9.7% 30-40%
+-  6.9% 40-50%
+
+Remember that the percentage reported above is inclusive at the next level, for example the studies with
+40-50% lost are also accounted for at lower levels.
```

Comment:
Maybe also include a comment that we did not find any strong patterns among the studies that were most greatly affected, whether they were from a specific sample type (according to empo category) or target 16S variable region.